lang/cs/Org.Apache.REEF.IO.Tests/TestAzureDataLakeFileSystemE2E.cs

```diff
@@ -194,6 +194,20 @@ public void TestCreateDirectoryE2E()
             Assert.True(_adlsClient.CheckExists(dirName));
         }
 
+        [Fact(Skip = SkipMessage)]
+        public void TestIsDirectoryE2E()
+        {
+            string dirName = $"/{_defaultFolderName}";
+            string fakeDirName = $"/fakeDir";
+
+            _adlsClient.CreateDirectory(dirName);
+            string fileName = UploadFromString(ContentsText);
+
+            Assert.True(_fileSystem.IsDirectory(PathToFile(dirName)));
+            Assert.False(_fileSystem.IsDirectory(PathToFile(fakeDirName)));
+            Assert.False(_fileSystem.IsDirectory(PathToFile(fileName)));
```

Comment:
We test for three different conditions here. Maybe split it into three unit tests?

Reply:
done

---
In reply to: [198681751](https://github.com/apache/reef/pull/1473#discussion_r198681751) [](ancestors = 198681751)